src/interface/misc.rs

```diff
@@ -198,6 +198,12 @@ pub fn lind_sigismember(set: SigsetType, signum: i32) -> bool {
     set & (1 << (signum - 1)) != 0
 }
 
+// Signals
+pub fn kill_inner(cage_id: u64, sig: i32) -> i32 {
```

Comment:
I see you moved this to the interface, which I think is a good idea. We don't usally use the "inner" name here though. Maybe rename this `lind_kill`?